cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -526,11 +534,42 @@
 
         logger.warn('-----> set status to %s', status)
 
-        job.write_dataserver_status(
-            status_dictionary_value=status, full_dict=self.par_dic)
+        if self.is_email_to_send_callback(status):
+            try:
+                self.send_email(status,
+                                instrument=instrument_name,
+                                time_request=time_request)
+                self.par_dic['mail_status'] = 'mail sent'
```

Comment:
you use `par_dict` to store outcome of the request processing? will it not cause issues in some places, e.g. since `par_dict` is used to build `job_id`? In general `par_dict` was meant to be more or less the *input parameter dict*.

Reply:
I thought about it, an alternative could be not to store anything, and just assume that the email was sent correctly. In case of problems, then additional information should be stored.

Reply:
What about storing this information inside the monitor of the job? In this way, the input list of parameter won't be affected, and the status of the email still logged